scripts/pre_commit_linter.py

```diff
@@ -2541,6 +2541,53 @@ def _check_npm_packages(self):
 
         return summary_messages
 
+    def _check_todo_comments_have_user(self):
+        """Checks that a todo comment has an assigned user
+        in the format TODO(@username): message.
+
+        Returns:
+            summary_messages: list(str). Summary of messages
+            generated by the check.
+        """
+        if self.verbose_mode_enabled:
+            print 'Starting todo comments check'
+            print '----------------------------------------'
+
+        summary_messages = []
+        message = 'Each todo comment should be assigned to a user'
+        # Match a regular expression that is independent of the comment format.
+        todo_regex = re.compile(r'TODO\s?\(@?[A-Z|a-z|\d]+\):?\s?.+')
```

Comment:
Can this be done just by adding the regex inside [BAD_PATTERNS_PYTHON_REGEXP ](https://github.com/oppia/oppia/blob/develop/scripts/pre_commit_linter.py#L267)? (not sure!)

Reply:
@DubeySandeep Yes! Much cleaner now. Thanks for the suggestion